### PR TITLE
Adds a colorable hood to the loadout.

### PR DESCRIPTION
**Pull request by ASmallCuteCat**
Adds a colorable hood to the loadout. Tested with human, unathi, and tajara.

Sprites by ASmallCuteCat (aka Revenant1367)

### CODE REVIEW

**Comment by ASmallCuteCat** [comment]
No idea how any of this works, flying by the seat of my pants here

!review

**Review by Llywelwyn — CHANGES_REQUESTED** [review]
not a dev, but
you'll need to get rid of the comments, and probably move the loadout datum up so it follows on after the other head items, but otherwise the code looks fine

File: code/modules/client/preference_setup/loadout/items/head.dm [hunk 1]
```diff
@@ -492,3 +492,10 @@ var/datum/gear_tweak/hair_block/gear_tweak_hair_block = new()
 			H.flags_inv |= BLOCKHEADHAIR
 		if("No")
 			H.flags_inv &= ~BLOCKHEADHAIR
+
+// REV EDITS OH BOY THESE ARE GONNA SUCK WTF AM I DOING
+/datum/gear/head/plain_hood
+	display_name = "colorable hood"
+	path = /obj/item/clothing/head/plain_hood
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
```
[the remark this answers is not in the document]
Comment (ASmallCuteCat) [reply code/modules/client/preference_setup/loadout/items/head.dm:500]: Code block moved, comments erased. Now to see if I can figure out how to update the github... :P

**Comment by ASmallCuteCat** [comment]
!wip

HELP THE THING KEEPS TRYING TO DELETE AURORASTATION.DME

**Comment by ASmallCuteCat** [comment]
ok i'm killing this and trying again

**Pull request closed without merging**